classes/Exception/ThankYouAuthor.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Claromentis\ThankYou\Exception;
+
+class ThankYouAuthor extends ThankYouException
```

Comment:
Could this be `AuthorNotFoundException`, or perhaps just `ThankYouAuthorException`? It's a bit unclear at the moment.

Reply:
Fixed in 0a715e95a8ea999877d173ffe95200591dbce78f (I removed it as it's no longer used, problem solved!)